### PR TITLE
Tweak: Fix typos [ED-15394]

### PR DESCRIPTION
**Pull request by szepeviktor**
## PR Checklist

- [ ] The commit message follows our guidelines:  https://github.com/elementor/elementor/blob/master/.github/CONTRIBUTING.md

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [x] Other... Please describe: fix misspellings, even in file names

## Summary

Found few misspellings.

https://github.com/crate-ci/typos

### CODE REVIEW

**Comment by szepeviktor** [comment]
Actually `$optin_url` is `$opt_in_url`
https://www.merriam-webster.com/dictionary/opt%20in

**Comment by toineenzo** [comment]
@szepeviktor Great catches! Hope they will merge your clean-up soon. It looks good.

**Review by toineenzo — APPROVED** [review]

**Comment by szepeviktor** [comment]
@davseve Could you merge this before got conflict goes boom? 💣

**Review by rami-elementor — CHANGES_REQUESTED** [review]

File: modules/nested-accordion/assets/js/frontend/handlers/nested-accordion-title-keyboard-handler.js [hunk 1]
```diff
@@ -27,7 +27,7 @@ export default class NestedAccordionTitleKeyboardHandler extends NestedTitleKeyb
 		};
 	}
 
-	handeTitleLinkEnterOrSpaceEvent( event ) {
+	handleTitleLinkEnterOrSpaceEvent( event ) {
```
[the remark this answers is not in the document]
Comment (szepeviktor) [reply modules/nested-accordion/assets/js/frontend/handlers/nested-accordion-title-keyboard-handler.js:30]: @rami-elementor All 3 were reverted.

**Review by rami-elementor — APPROVED** [review]

**Comment by rami-elementor** [comment]
@matipojo @shilo-ey Please review AI-related changes. The PR renames the components folder.

**Review by matipojo — APPROVED** [review]

**Comment by yotamselementor** [comment]
Thanks @szepeviktor, this is much appreciated!

**Comment by szepeviktor** [comment]
Glad to contribute.

**Pull request merged**